src/Api/ProductMediaFileApi.php

```diff
@@ -105,4 +109,29 @@ public function download($code)
     {
         return $this->resourceClient->getStreamedResource(static::MEDIA_FILE_DOWNLOAD_PATH, [$code]);
     }
+
+    /**
+     * Extracts the code of a media-file from a creation response.
+     *
+     * @param ResponseInterface $response
+     *
+     * @throws \RuntimeException if unable to extract the code
+     *
+     * @return mixed
+     */
+    protected function extractCodeFromCreationResponse(ResponseInterface $response)
+    {
+        $headers = $response->getHeaders();
+
+        if (!isset($headers['Location'][0])) {
+            throw new \RuntimeException('The response does not contain the URI of the created media-file.');
```

Comment:
it means if a media has not been created, our client will send this response ? or maybe error is catched before ?